# ğŸª‘grayscale_ğŸ¬key_config
#	ãƒ‡ã‚£ãƒ¬ã‚¯ã‚¿ãƒ¼ï¼ˆDirectorï¼‰
#
#	ç´°ã‹ãªè¨­å®šã¯ä½•ã‚‚ã§ããªã„ã®ã§ã€ã‚½ãƒ¼ã‚¹ã‚’ç›´æ¥ã‚«ã‚¹ã‚¿ãƒã‚¤ã‚ºã™ã‚‹ã“ã¨
extends Node2D


# å€¤ã¯ãƒœã‚¿ãƒ³ç•ªå·ã€‚ãƒ¬ãƒãƒ¼ã¯ +1000
var key_config = {
	# ä»®æƒ³ã‚­ãƒ¼ï¼ˆï¼‘ï¼‰æ±ºå®šãƒœã‚¿ãƒ³ã€ãƒ¡ãƒƒã‚»ãƒ¼ã‚¸é€ã‚Šãƒœã‚¿ãƒ³
	&"VK_Ok" : -1,
	# ä»®æƒ³ã‚­ãƒ¼ï¼ˆï¼’ï¼‰ã‚­ãƒ£ãƒ³ã‚»ãƒ«ãƒœã‚¿ãƒ³ã€ãƒ¡ãƒ‹ãƒ¥ãƒ¼ãƒœã‚¿ãƒ³
	&"VK_Cancel" : -1,
	# ä»®æƒ³ã‚­ãƒ¼ï¼ˆï¼“ï¼‰ãƒ¡ãƒƒã‚»ãƒ¼ã‚¸æ—©é€ã‚Šãƒœã‚¿ãƒ³
	&"VK_FastForward" : -1,
}


# Called when the node enters the scene tree for the first time.
func _ready():
	# ã‚­ãƒ¼ãƒ»ã‚³ãƒ³ãƒ•ã‚£ã‚°ç”»é¢ã‚’å§‹ã‚ã‚‹ã‚¿ã‚¤ãƒŸãƒ³ã‚°ã§ä»¥ä¸‹ã‚’å‘¼å‡ºã™
	$"KeyConfigArtist".entry()


# Called every frame. 'delta' is the elapsed time since the previous frame.
func _process(delta):
	# ã‚­ãƒ¼ãƒ»ã‚³ãƒ³ãƒ•ã‚£ã‚°ã§å¿…è¦
	$"KeyConfigArtist".on_process(delta)


func _unhandled_input(event):
	# ã‚­ãƒ¼ãƒ»ã‚³ãƒ³ãƒ•ã‚£ã‚°ã§å¿…è¦
	$"KeyConfigArtist".on_unhandled_input(event)


func on_exit():
	print("ï¼»ã‚­ãƒ¼ã‚³ãƒ³ãƒ•ã‚£ã‚°ï¼½ã€€å®Œäº†")
